app/models/izakaya.py

```diff
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
+class Izakaya(BaseModel):
+    id: int
+    name: str
+    lng: float
+    lat: float
+    area: str
+    distance: float = None
```

Comment:
明示的な型定義をして欲しいのでここはOptionalを使用して欲しいです！
Noneを使用する場合は２つあって
- デフォルト値をNoneとして値を受け入れて, データを受け取れば値をセットする
- デフォルト値をなんらかのデータで受け取って選択肢にNoneを許容する

今回の場合は前者なのでデフォルト値がNoneで値をセットすることを明示的に示す`Optional`クラスを使用してほしいです
```suggestion
    distance: Optional[float] = None
```

Reply:
確かにこっちの方がわかりやすいかも
感謝！